README.md

```diff
@@ -17,7 +17,9 @@ extern "C" __global__ void set(int *a) {
 }
 """
 prog = hiprtc.hiprtcCreateProgram(source, 'set', [], [])
-hiprtc.hiprtcCompileProgram(prog, ['--offload-arch=gfx906'])
+device_properties = hip.hipGetDeviceProperties(0)
+print(f"Compiling kernel for {device_properties.name}")
```

Comment:
It should be gcnArchName. Name is empty for some devices

Reply:
Fixed